app/controllers/presentations_controller.rb

```diff
@@ -11,6 +11,7 @@ def show
 
   def new
     @presentation = Presentation.new
+    @event = Event.find(params[:event_id]) if params[:event_id]
```

Comment:
I think you can remove the conditional here. If `params[:id]` is `nil` then it won't be able to find an Event and `@event` ends up being `nil`, too.

* pretty sure